src/decider/agent/DeciderAgent.php

```diff
@@ -149,6 +159,14 @@ function processUploadId($uploadId)
     $this->activeRules = array_key_exists('r', $args) ? intval($args['r']) : self::RULES_ALL;
     $this->licenseMap = new LicenseMap($this->dbManager, $this->groupId, $this->licenseMapUsage);
 
+    if (array_key_exists("r", $args) && (($this->activeRules&self::RULES_COPYRIGHT_FALSE_POSITIVE)== self::RULES_COPYRIGHT_FALSE_POSITIVE)) {
+      $this->getCopyrightsToDisableFalsePositives($uploadId);
+    }
+    
```

Comment:
Hello @Kaushl2208,
PHP Codesniffer test is failing. You have not indented the code properly. Please take a look at the logs https://github.com/fossology/fossology/pull/2083/checks?check_run_id=3374035897#step:4:206

Reply:
Hey @avinal, Thanks for your review! This is work in progress and I thought first as the implementation will complete, I will clean and clear the check errors as well !! :smile:

Reply:
Yeah, ok